docs-conceptual/azps-7.3.2/az-predictor.md

```diff
@@ -0,0 +1,118 @@
+---
+description: Using Az Predictor for intelligent context-aware command completion in Azure PowerShell.
+ms.custom: devx-track-azurepowershell
+ms.date: 03/21/2022
+ms.devlang: powershell
+ms.service: azure-powershell
+ms.topic: conceptual
+title: Intelligent context-aware command completion with Az Predictor
+---
+
+# Intelligent context-aware command completion with Az Predictor
+
+## Overview
+
+[Az Predictor](https://www.powershellgallery.com/packages/Az.Tools.Predictor/) is a PowerShell
+module that helps you navigate the cmdlets and parameters of the
+[Az PowerShell module](https://www.powershellgallery.com/packages/Az). It provides intelligent
+context-aware suggestions for command completion when using Azure PowerShell.
+
+Az Predictor uses the
+[subsystem plugin model](/powershell/scripting/learn/experimental-features#pssubsystempluginmodel)
+available in PowerShell 7.2. This updated version requires
+[PSReadLine 2.2.2](https://www.powershellgallery.com/packages/PSReadLine/2.2.2) or higher to display
+suggestions.
+
+## Requirements
+
+Required configuration for Az Predictor:
+
+- [PowerShell 7.2](https://github.com/PowerShell/PowerShell/) or higher
+- [PSReadline 2.2.2](https://github.com/PowerShell/PSReadLine/) or higher
+
+Install the latest version of PSReadLine:
+
+```powershell
+Install-Module -Name PSReadline
+```
+
+## Getting started
+
+### Install Az Predictor
+
+Install the Az.Tools.Predictor PowerShell module
+
+```powershell
+Install-Module -Name Az.Tools.Predictor -AllowPrerelease
+```
+
+### Enable Az Predictor
+
+1. Enable Az Predictor for the current and future PowerShell sessions.
+
+   ```powershell
+   Enable-AzPredictor -AllSession
+   ```
+
+1. Set your preferred view for suggestions.
+
+   Enable list view:
```

Comment:
Can we add a screen capture to explain list view? It would be helpful to also have a link to the official documentation for list view